examples/deprecated/opssight.json

```diff
@@ -6,7 +6,7 @@
   },
   "spec": {
     "namespace": "opssight-test",
-    "desiredState": "start",
+    "desiredState": "",
```

Comment:
I think opssight won't work with empty desiredState.. I think @yashbhutwala need to revert the change